lib/contracts/withdraw.go

```diff
@@ -0,0 +1,52 @@
+package contracts
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/omni-network/omni/lib/netconf"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/params"
+
+	"cosmossdk.io/math"
+)
+
+type WithWithdrawThreshold struct {
+	Name        string
+	OnlyOmniEVM bool
+	NotOmniEVM  bool
+	Address     common.Address
+	Thresholds  WithdrawThresholds
+}
+
+func ToWithdraw(ctx context.Context, network netconf.ID) ([]WithWithdrawThreshold, error) {
+	addrs, err := GetAddresses(ctx, network)
+	if err != nil {
+		return nil, err
+	}
+
+	return []WithWithdrawThreshold{
+		{
+			Name:        "gas-pump",
+			Address:     addrs.GasPump,
+			OnlyOmniEVM: false,
+			NotOmniEVM:  true,
+			Thresholds:  WithdrawThresholds{minEther: 10},
+		},
+	}, nil
+}
+
+type WithdrawThresholds struct {
+	minEther float64
```

Comment:
This would be `maxEther` for withdraw right?

Reply:
I was reading it like "minimum Ether required for withdrawal" but that perspective works too